docker/Dockerfile

```
@@ -0,0 +1,55 @@
+RUN ls Resources/Views
+
+
+FROM --platform=$BUILDPLATFORM swift:6.0.3-noble AS server-build
+
+# Cross compilation based on https://swiftinit.org/articles/cross-compiling-x86_64-linux-to-aarch64-linux
+RUN apt update && apt -y install gcc-aarch64-linux-gnu wget2 && apt -y install g++-multilib
+RUN mkdir /home/swift-cross-compile /home/vapor
+WORKDIR /home/swift-cross-compile
+COPY docker/swift-cross-platform/* /home/swift-cross-compile/
+
+RUN export DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true \
```

Hadolint (2.12.0)

[error] Do not use apt-get dist-upgrade

(DL3005)